front/pages/api/w/[wId]/assistants/[aId]/index.ts

```diff
@@ -11,6 +18,74 @@ export type GetAssistantResponseBody = {
   assistant: AgentConfigurationType;
 };
 
+const PatchAssistantResponseBodySchema = t.type({
```

Comment:
Should we just reuse the time from the creation endpoint?

Reply:
I added a commit!